fastlane/Fastfile

```diff
@@ -185,22 +185,33 @@ end
 
 desc "Pushes the StreamChat and StreamChatUI SDK podspecs to Cocoapods trunk"
 lane :push_pods do |options|
-  # First pod release will not have any problems
-  pod_push(path: "StreamChat.podspec", allow_warnings: true)
 
-  def release_ui(sync)
+  def release_pod(sync:, podspec:, dry:)
     begin
-      pod_push(path: "StreamChatUI.podspec", allow_warnings: true, synchronous: sync)
-    rescue
-      puts "pod_push failed. Waiting a minute until retry for trunk to get updated..."
+      UI.message "Starting to push podspec: #{podspec}"
+
+      if dry == false
```

Comment:
This function does nothing when dry == true? What does dry and sync mean?  Would be nice to add a bit of documentation here

Reply:
Ok, I see it now in the PR description. But would be nice to have it in the code as well

Reply:
Ill add that ;)

Reply:
Done! Please re-review @polqf